### PR TITLE
feat: Auto create Quarkus run config while importing project

### LINKED ISSUE #1233: Auto create Quarkus run config while importing project
When a Quarkus Maven / Gradle project is imported, we should auto create a Quarkus run config. It will give the capability to fill the standard IJ Services view with the auto created run config.

This auto create should be done according a settings from a Quarkus page settings.

### PR DESCRIPTION
**Pull request by angelozerr**
feat: Auto create Quarkus run config while importing project

Fixes #1233

### CODE REVIEW

File: src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfigurationManager.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+ *
+ * <ul>
+ *     <li>auto create Quarkus run configuration when a Quarkus Maven/Gradle is imported</li>
+ *     <li>provides methods to find existing/ create Quarkus run configuration from a given module</li>
+ * </ul>
+ */
+public class QuarkusRunConfigurationManager implements Disposable {
+
+    public static QuarkusRunConfigurationManager getInstance(Project project) {
+        return ServiceManager.getService(project, QuarkusRunConfigurationManager.class);
```
Comment (fbricon) [inline src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfigurationManager.java:48]:
```java
return project.getService(project, QuarkusRunConfigurationManager.class);
```

Comment (fbricon) [reply src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfigurationManager.java:48]: See https://github.com/redhat-developer/intellij-quarkus/pull/1258/files

File: src/main/resources/messages/QuarkusBundle.properties [hunk 2]
```diff
@@ -11,6 +11,11 @@
 #     Red Hat Inc. - initial API and implementation
 ###############################################################################
 
+## Quarkus UI settings page
+quarkus=Quarkus
+quarkus.title=Quarkus configuration
+quarkus.create.quarkus.run.configuration.while.project.import=Create Quarkus run configuration while project import?
```
Comment (fbricon) [inline src/main/resources/messages/QuarkusBundle.properties:17]: Create "Quarkus Dev Mode" run configuration on project import

Comment (fbricon) [reply src/main/resources/messages/QuarkusBundle.properties:17]: no `?`

Comment (angelozerr) [reply src/main/resources/messages/QuarkusBundle.properties]: fixed

File: src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfigurationManager.java [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+        }
+        if (runManager.getAllSettings().size() == 1) {
+            // Select the Quarkus configuration on the top of the Run configuration list
+            runManager.setSelectedConfiguration(quarkusSettings);
+        }
+        return quarkusSettings;
+    }
+
+    @NotNull
+    private static String generateConfigurationName(@NotNull Module module, RunManager runManager) {
```
Comment (fbricon) [inline src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfigurationManager.java:94]: you can delete that method now

Comment (angelozerr) [reply src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunConfigurationManager.java]: fixed

**Comment by fbricon** [comment]
@angelozerr awesome, thanks!

**Pull request merged**